installation/sql/postgresql/extensions.sql

```diff
@@ -907,3 +907,28 @@ BEGIN
   RETURN soundex;
 END;
 $$;
+
+--
+-- Table structure for table "#__cookiemanager_cookies"
+--
+
+CREATE TABLE IF NOT EXISTS "#__cookiemanager_cookies" (
+  "id" serial NOT NULL,
+  "title" varchar(255) NOT NULL,
+  "alias" varchar(400) NOT NULL,
+  "cookie_name" varchar(255) NOT NULL,
+  "cookie_desc" varchar(255) NOT NULL,
+  "exp_period" varchar(20) NOT NULL,
+  "exp_value" integer DEFAULT 0 NOT NULL,
+  "catid" integer DEFAULT 0 NOT NULL,
+  "published" smallint DEFAULT 1 NOT NULL,
+  "ordering" integer DEFAULT 0 NOT NULL,
+  "created" timestamp without time zone NOT NULL,
+  "created_by" integer DEFAULT 0 NOT NULL,
+  "modified" timestamp without time zone NOT NULL,
+  "modified_by" integer DEFAULT 0 NOT NULL,
+  PRIMARY KEY ("id")
+);
+CREATE INDEX "#__cookiemanager_cookies_idx_state" on "#__cookiemanager_cookies" ("published");
+CREATE INDEX "#__cookiemanager_cookies_idx_catid" on "#__cookiemanager_cookies" ("catid");
+CREATE INDEX "#__cookiemanager_cookies_idx_createdby" on "#__cookiemanager_cookies" ("created_by");
```

Comment:
@rjharishabh I found the problem: You have added the above code to the end of the extensions.sql for PostgreSQL.

If you compare both files extensions.sql you will see that only the file for PostgreSQL has a `CREATE OR REPLACE FUNCTION soundex(input text) RETURNS text ...`. This has to be at the end of the file, but you have placed your create table statement below that.

If you move the code marked above to before that soundex stuff so that your create table comes after the one for the `#__action_logs_users` table, it will solve the issue and also make that file consistent with the MySQL file (cookies table after action logs table.

Reply:
In the previous commit, I have added  this before this function,
but I think I have to add after this because it's been added after that.

Reply:
What? No. It has to be before the soundex. The soundex has to be the last thing in that file. You have seen the PostgreSQL system tests were failing in drone with your stuff at the end, and now as your thing comes before the soundex they will pass.